designer/server/src/models/forms/create.js

```diff
@@ -11,7 +11,7 @@ import {
  * @param {ValidationFailure} [validation]
  */
 export function titleViewModel(metadata, validation) {
-  const pageTitle = 'Enter a name for your form'
+  const pageTitle = 'Form title'
```

Comment:
is there a reason you changed this?

Reply:
I thought I have seen that mentioned in the ticket or prototype but I am not sure now. I will revert it back.